src/backup/abstract_should_backup_predicate.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from abc import abstractmethod
+
+
+class AbstractShouldBackupPredicate(object):
```

Comment:
Can we remove Should from `AbstractShouldBackupPredicate` and `OnDemandShouldBackupPredicate`

Reply:
done